Update direct-routing-plan.md (#4169)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/direct-routing-plan.md

  > [!NOTE]
  > Microsoft recommends at least two ports per concurrent call on the SBC.

> ## Media traffic: Media Processors geography

Reviewer: ```suggestion
## Media traffic: Media processors geography
```

> The media traffic flows via components, called Media Processors. 

Reviewer: ```suggestion
The media traffic flows via components called media processors. 
```

> Media Processors placed in the same datacenters as SIP Proxies. Also, there are additional media processors to optimize the media flow. For example, now we do not have a SIP Proxy component in Australia (SIP flows via Singapore or Hong Kong) but we do have the Media Processor locally in Australia. Need of the Media Processors locally dictated by the latency which we  experience by sending traffic on long distance, for example from Australia to Singapore or Hong Kong. While latency in the example of traffic flowing from Australia to Hong Kong or Singapore is acceptable to preserve good call quality for SIP traffic, for Real Time Media traffic it is not.

Reviewer: ```suggestion
Media processors are placed in the same datacenters as SIP proxies. Also, there are additional media processors to optimize media flow. For example, now we do not have a SIP proxy component in Australia (SIP flows via Singapore or Hong Kong) but we do have the media processor locally in Australia. The need for the media processors locally is dictated by the latency which we experience by sending traffic long-distance, for example from Australia to Singapore or Hong Kong. While latency in the example of traffic flowing from Australia to Hong Kong or Singapore is acceptable to preserve good call quality for SIP traffic, for real-time media traffic it is not.
```

> Location of the Media Processors:

Reviewer: ```suggestion
Location of the media processors:
```

> Locations where both SIP proxy and Media Processor components deployed:

Reviewer: ```suggestion
Locations where both SIP proxy and media processor components deployed:
```

> 	- US (two in US west and US East datacenters)

Reviewer: ```suggestion
	- US (two in US West and US East datacenters)
```

	- Europe (Amsterdam and Dublin datacenters)
	- Asia (Singapore and Hong Kong datacenters)

> Locations where only Media Processors deployed (SIP flows via nearest datacenter listed above)

Reviewer: ```suggestion
Locations where only media processors are deployed (SIP flows via the closest datacenter listed above):
```
